### DISCUSSION TITLE
Why were the patch versions for the vulnerability (snyk id: SNYK-GOLANG-GITHUBCOMMOHAMMED90CADDYSSHINTERNALAUTHENTICATIONOS-3032989) released so late?

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [mohammed90/caddy-ssh#21] [Q&A] [answered]
Hello, we are a research team working on Golang. During our investigation, we found vulnerability(https://security.snyk.io/vuln/SNYK-GOLANG-GITHUBCOMMOHAMMED90CADDYSSHINTERNALAUTHENTICATIONOS-3032989) was addressed in commit e7bdc1fd2d63f0ecfaa64059e81ce92cd91c68c8. However, we noticed that the patch version was released after long time (234 days).  We are curious about the reasons behind the delayed release of the patch version, as it may hinder the efficient distribution of patches to downstream users. Could the reason be 

1.Issues with testing and CI checking. 

2.Other commits have to be incorporated into one release. 

3.By convention, versions are not frequently released. 

4.Other reasons. 

Thank you for your attention, and we look forward to receiving your reply.

**Comment by francislavoie** [comment, marked as answer]
Notice that this is a v0, entirely experimental project. Expecting any kind of timely releases is kinda ridiculous. I don't understand where these expectations are coming from.

**Comment by mohammed90** [comment]
Adding to what Francis said, while my project isn't holding up that much infrastructure yet, but I'm a lone developer trying to balance this project, full-time job, school, life, and other responsibilities...

![image](https://github.com/kadeessh/kadeessh/assets/2636183/a3ac0b31-117f-4ef0-9e47-07ac0adc5fdf)

[Source](https://m.xkcd.com/2347/)

**Reply by Silence-worker-02** [reply]
I sincerely apologize for any inconvenience caused. Our aim is to conduct a survey regarding the delayed release of versions by maintainers.
